pkg/network/protocols/http/statkeeper.go

```diff
@@ -82,6 +86,13 @@ func (h *StatKeeper) add(tx Transaction) {
 		h.telemetry.emptyPath.Add(1)
 		return
 	}
+
+	// Quantize HTTP path
+	// (eg. this turns /orders/123/view` into `/orders/*/view`)
+	if h.cfg.EnableUSMQuantization {
+		rawPath = h.quantizer.Quantize(rawPath)
+	}
```

Comment:
We should not run it for gRPC protocol

Reply:
what about HTTP2?

Reply:
sorry, grpc and http2 are the same for me in the context of USM